en/concepts/scene/light/lightType/spot-light.md

```diff
@@ -1,26 +1,30 @@
 # Spotlights
```

Comment:
```suggestion
# Spotlight
```

Reply:
In Unity's documentation, they use plural in the title of all types of light except for ambient light.